wagtail/admin/views/pages.py

```diff
@@ -672,8 +672,9 @@ def get(self, request, *args, **kwargs):
 
 class PreviewOnCreate(PreviewOnEdit):
     def get_page(self):
-        (content_type_app_name, content_type_model_name,
-         parent_page_id) = self.args
+        content_type_app_name = self.kwargs["content_type_app_name"]
+        content_type_model_name = self.kwargs["content_type_model_name"]
+        parent_page_id = self.kwargs["parent_page_id"]
```

Comment:
I changed this because `self.args` is empty now (idk why?). We can read the variables from `self.kwargs` but not in the same way it used be.

Reply:
That makes sense - changing the URL path has changed these from positional arguments to named arguments (and with this being a class-based view, we don't have a function definition to persuade Python to treat them as positional again).